src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let totalCost = days * 40;
+
+  if (days >= 7) {
+    totalCost -= 50;
+  } else if (days >= 3) {
+    totalCost -= 20;
```

Comment:
Also, you **could/might** use a direct return after the conditionals

[and remember checklist 4 and 5](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md)

Reply:
Wow, i'm guessing i dind't saw that. Fixed already, following the checklist now.